restapi/src/main/java/io/stargate/web/docsapi/service/ReactiveDocumentService.java

```diff
@@ -381,9 +463,10 @@ public ReactiveDocumentService(
       String collection,
       String documentId,
       String payload,
+      Integer ttl,
```

Comment:
OK this is a mistake, how can a patch have a ttl as int? should be a `ttlAuto`? This also means you don't have to introduce that new `patchSubDocument` it can still be one method.. Right?

Reply:
I believe this method is entirely unused, that's why I missed it. Deleting it in favor of the sub-document impl and renaming that method